namer/consul/src/main/scala/io/buoyant/namer/consul/package.scala

```diff
@@ -1,10 +1,26 @@
 package io.buoyant.namer
 
+import com.twitter.finagle.Path
 import com.twitter.logging.Logger
 import com.twitter.util.{Activity, Updatable, Var}
 
 package object consul {
   private[consul]type VarUp[T] = Var[T] with Updatable[T]
   private[consul]type ActUp[T] = VarUp[Activity.State[T]]
   val log = Logger.get("io.buoyant.namer.consul")
+
+  /** wraps a path and keeps additional consul-specific structure */
+  private[consul] case class ConsulPath(
+    raw: Path,
+    scheme: Option[PathScheme] = None
```

Comment:
Why is this an Option?  As far as I can tell it is always Some.

Reply:
Now that I think of it, after the latest changes it doesn't makes sense anymore, I'll fix